domains/auth/Auth.API/API/Services/OidcProviders/SignaturGruppen.cs

```diff
@@ -37,4 +41,19 @@ public NextStep CreateAuthorizationUri(AuthState state)
 
         return authorizationUri;
     }
+
+    public async Task Logout(string token)
+    {
+        var url = authOptions.OidcUrl;
+        httpClient.BaseAddress = new Uri(url);
+
+        var response = await httpClient.PostAsJsonAsync("/api/v1/session/logout", new { id_token = token });
+        if (!response.IsSuccessStatusCode)
+        {
+            var content = await response.Content.ReadAsStringAsync();
+
+            logger.LogWarning("StatusCode: {StatusCode}, url: {Url}, content: {Content}",
+                response.StatusCode, response.RequestMessage?.RequestUri, content);
```

Comment:
This will be fixed when we run an editorconfig in [762](https://app.zenhub.com/workspaces/fenris---team-board-616bc40121d71900140955f8/issues/energinet-datahub/energy-origin-issues/762)